application/frontend/bnd.bnd

```diff
@@ -0,0 +1,2 @@
+Sling-Bundle-Resources: /apps/luna/webroot
```

Comment:
Let's change it to `/apps/luna/web_root;/apps/luna/author` - we will install resources from those paths

Also, let's go to the `application/backend/bnd.bnd` and replace content with
```
Sling-Bundle-Resources: /apps/luna/components
``` 
It would prevent overriding resources